packages/config/config/devices/0x027a/zen16.json

```
@@ -74,6 +74,13 @@
 			"$import": "templates/zooz_template.json#zen16_switch_type_v2",
 			"label": "Switch 1: Type"
 		},
+		{
+			"#": "25",
+			"$if": "firmwareVersion >= 1.2",
```

```suggestion
			"$if": "firmwareVersion >= 2.0",
```
Same for the other new parameters.